features/networking/sgw-lgw-migration.feature

```diff
@@ -177,3 +177,80 @@ Feature: SGW<->LGW migration related scenarios
     When I run commands on the host:
       | curl --connect-timeout 5 <%= cb.hostip %>:<%= cb.port %> |
     Then the step should fail
+
+  # @author weliang@redhat.com
+  # @case_id OCP-48066
+  @4.10
+  @admin
+  @destructive
+  @network-ovnkubernetes
+  @vsphere-ipi
+  @baremetal-upi
+  Scenario: [SDN-2290] SGW <-> LGW migration scenarios for externalIP	
+    Given the env is using "OVNKubernetes" networkType
+    ######## Prepare Data Pre Migration for multiple use cases############
```

Comment:
@weliang1 LGTM all except
Since its a single scenario and we plan to keep it separate due to its complexity, it miht be enough to mention
```##### Prepare Data Pre Migration #### and #### Check Data Post Migration ####```

Thanks

Reply:
@anuragthehatter Just update both. Thanks!